packages/core/src/tracing/sampling.ts

```diff
@@ -41,7 +41,12 @@ export function sampleSpan(
   const parsedSampleRate = parseSampleRate(sampleRate);
 
   if (parsedSampleRate === undefined) {
-    DEBUG_BUILD && logger.warn('[Tracing] Discarding transaction because of invalid sample rate.');
+    DEBUG_BUILD &&
+      logger.trace(
```

Comment:
l: why use `logger.trace` here vs. `logger.log` or similar?

Reply:
whoops, I used trace to debug. Switching back... Thanks!